wvd-templates/Create and provision WVD host pool/nestedtemplates/unmanagedDisks-galleryvm.json

```diff
@@ -157,9 +168,7 @@
                 "hardwareProfile": {
                     "vmSize": "[parameters('rdshVmSize')]"
                 },
-                "availabilitySet": {
-                    "id": "[resourceId('Microsoft.Compute/availabilitySets/', concat(parameters('rdshPrefix'), 'availabilitySet'))]"
-                },
+                "availabilitySet": "[if(equals(parameters('enablePersistentDesktop'),bool('False')),variables('availabilitySetId'),json('null'))]",
```

Comment:
This seems redundant...if you're already doing the if/else for the "availabilitySetId" variable, can you just pass that in to "availabilitySet" here, instead of doing another if/else?

Reply:
yep, that makes sense, I'll fix it.

Reply:
BTW, I found out why I did that way, there is no way to hide the id attribute without this extra logic and ARM does not accept an id=null value.

Reply:
Got it. Sounds good.